docs/pages/ESSupport.md

```diff
@@ -0,0 +1,368 @@
+
+## ECMAScript Version Support
+
+The UI5 Tooling offers general support for the latest ECMAScript features. While a `ui5 build` is executed, the UI5 Tooling analyses the code in a few places. Depending on your project you have to consider some restrictions regarding the used ECMAScript syntax.
+
+## General
+The following sections describe the limitations valid for all types of projects.
```

Comment:
General sentence that using Object Expressions, Spread Elements, and Template Literals with expressions might lead to limitations.

Reply:
achieved by new structure